assets/css/main.css

```diff
@@ -7,7 +7,7 @@
  */
 
 body {
-	@apply text-white;
+	@apply font-kumbhsans text-white;
```

Comment:
There is no need to add this globally to body. We should call kumbhsans only when needed.